tensorflow_lite_support/tools/Build_TensorFlow_Lite_Support_Libraries_with_Bazel_and_Colab.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    !wget -q 'https://dl.google.com/android/repository/tools_r25.2.5-linux.zip'
</pre><p> </p><p> </p><p>Does it work on Windows if we download the SDK and NDK for Windows?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tflite-support/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I have not tried that on windows.

Reply:
OK. No worries!